lib/util.js

```diff
@@ -761,8 +761,12 @@ function formatError(value) {
 function formatObject(ctx, value, recurseTimes, keys) {
   const len = keys.length;
   const output = new Array(len);
-  for (var i = 0; i < len; i++)
-    output[i] = formatProperty(ctx, value, recurseTimes, keys[i], 0);
+  const isNs = types.isModuleNamespaceObject(value);
```

Comment:
This line should rather go along with the value checks at line 571.

Reply:
i'm not quite sure how to do that, should i make another formatObject function like formatNamespaceObject?

Reply:
Yes, that would be the ideal way here. That way there is no performance impact on regular objects.

Reply:
I am not sure how V8 optimizes the loop in this case. I guess it could be smart to remove the `isNs` check but otherwise there would be a minor overhead.